enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/discovery/TopologyService.java

```diff
@@ -31,6 +31,7 @@
  */
 public interface TopologyService extends Lifecycle
 {
+
```

Comment:
:grey_question:

Reply:
Force of habit I think. Fixed and will keep an eye